### DISCUSSION TITLE
Excluding traffic

### DISCUSSION THREAD

**Discussion by dmudro** [plausible/analytics#2006] [Anything Else]
the general advise for excluding internal traffic is using [ad blockers](https://plausible.io/docs/excluding). 

my quick idea is having the ability drop something in the hosted HTML implementing Plausible script tag to disable tracking for that session. could be global js var.

haven't thought this trough completely but hopefully the underlying idea is clear: having the ability to disable Plausible in our SSR in order to stop polluting stats with our internal traffic. a couple of use cases come to my mind:
- in our backend, I could identify our internal users logged in on our site with @example.com email address
- could sniff for an internal IP address in our backend
- basically any custom backend logic disabling tracking via SSR + JS var before its fired

or internal traffic can be quite large and pollutes analytics significantly. ad blockers not realistic in a team of <50 generating internal traffic.

thanks for considering!

**Comment by metmarkosaric** [comment]
Thanks for the feedback @dmudro! Currently, you can also set a special localStorage flag in the browser which is easier if you want to exclude more people. Take a look at this https://plausible.io/docs/excluding-localstorage

**Reply by dmudro** [reply]
awesome, I missed this doc. many thanks @metmarkosaric 

glad to have reinvented the wheel in my suggestion above :)

I might just code backend around generating the localStorage piece.
